README.md

```diff
@@ -19,6 +19,120 @@ let package = Package(
 )
 ```
 
+## Example
+In this example we'll build a simple "reverse-echo" client and server that utilizes a secure SSL socket for sending and receiving messages.
+
+### Client
+The client connects to a server available at `serverFQDN:8443` and sends a string.  It then waits for a response from the server and then exits.
+
+```swift
+import TCPSSL
+
+guard Process.arguments.count == 3 else {
+  print("Usage:  client serverFQDN string")
+  exit(0)
+}
+
+let serverFQDN = Process.arguments[1]
+let sendData   = Data(Process.arguments[2])
+
+do {
```

Comment:
Since this is at the top level anyway, there's no reason to catch and handle the error manually - Swift will do that for you 😄